content/ko/docs/concepts/workloads/controllers/deployment.md

```diff
@@ -0,0 +1,1144 @@
+    nginx-deployment-1564180365-nacti   1/1       Running   0          14s
+    nginx-deployment-1564180365-z9gth   1/1       Running   0          14s
+    ```
+
+    다음에 이러한 파드를 업데이트 하려면 디플로이먼트의 파드 템플릿만 다시 업데이트 하면 된다.
+
+    디플로이먼트는 업데이트되는 동안 일정한 수의 파드만 중단되도록 보장한다. 기본적으로
+    의도한 파드의 수 중 25% 이상이 되도록 보장한다(최대 25% 이상 불가).
+
+    또한 디플로이먼트는 의도한 수의 파드 이상으로 파드가 생성되도록 보장한다.
+    기본적으로 의도한 파드의 수의 최대 25%가 되도록 보장한다(최대 25% 까지).
+
+    예를 들어, 위 디플로이먼트를 자세히 살펴보면 먼저 새로운 파드를 생성한 다음 
+    이전 파드를 삭제하고, 새로운 파드를 만든 것을 볼 수 있다. 새로은 파드가 나올 때까지 이전 파드를 죽이지 않으며, 
+    이전 파드가 충분히 죽기 전까지 새로운 파드를 만들지 않는다.
+    이것은 최소 2개의 파드를 사용할 수 있게 하고, 최대 4개의 파드를 사용할 수 있게 한다.
+
+* 디플로이먼트의 세부 정보 가져오기
+  ```shell
+  kubectl describe deployments
+  ```
+  이와 유사하게 출력된다. 
+  ```
+  Name:                   nginx-deployment
+  Namespace:              default
+  CreationTimestamp:      Thu, 30 Nov 2017 10:56:25 +0000
+  Labels:                 app=nginx
+  Annotations:            deployment.kubernetes.io/revision=2
+  Selector:               app=nginx
+  Replicas:               3 desired | 3 updated | 3 total | 3 available | 0 unavailable
+  StrategyType:           RollingUpdate
+  MinReadySeconds:        0
+  RollingUpdateStrategy:  25% max unavailable, 25% max surge
+  Pod Template:
+    Labels:  app=nginx
+     Containers:
+      nginx:
+        Image:        nginx:1.9.1
+        Port:         80/TCP
+        Environment:  <none>
+        Mounts:       <none>
+      Volumes:        <none>
+    Conditions:
+      Type           Status  Reason
+      ----           ------  ------
+      Available      True    MinimumReplicasAvailable
+      Progressing    True    NewReplicaSetAvailable
+    OldReplicaSets:  <none>
+    NewReplicaSet:   nginx-deployment-1564180365 (3/3 replicas created)
+    Events:
+      Type    Reason             Age   From                   Message
+      ----    ------             ----  ----                   -------
+      Normal  ScalingReplicaSet  2m    deployment-controller  Scaled up replica set nginx-deployment-2035384211 to 3
+      Normal  ScalingReplicaSet  24s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 1
+      Normal  ScalingReplicaSet  22s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 2
+      Normal  ScalingReplicaSet  22s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 2
+      Normal  ScalingReplicaSet  19s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 1
+      Normal  ScalingReplicaSet  19s   deployment-controller  Scaled up replica set nginx-deployment-1564180365 to 3
+      Normal  ScalingReplicaSet  14s   deployment-controller  Scaled down replica set nginx-deployment-2035384211 to 0
+    ```
+    여기에서 처음 디플로이먼트를 생성할 때 레플리카셋(nginx-deployment-2035384211)을 생성해서 
+    3개의 레플리카를 직접 스케일 업한 것을 볼 수 있다.
+    디플로이먼트를 배포할 때 새 레플리카셋(nginx-deployment-1564180365)을 생성하고, 1개로 스케일업한 다음 
```

Comment:
When you **updated** the Deployment, it created a new ReplicaSet
    (nginx-deployment-1564180365) and scaled it up to 1 and then scaled down the old ReplicaSet to 2, so that at
    least 2 Pods were available and at most 4 Pods were created at all times. 

```suggestion
    디플로이먼트를 업데이트할 때 새 레플리카셋(nginx-deployment-1564180365)을 생성하고, 1개로 스케일업한 다음 
```